Week3/homework/3-step3.js

```diff
@@ -2,24 +2,37 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+  
+  let answerStr = "";
+  for(let i = 0; i < num; i++) {
+    answerStr += str;
+  }
+  return answerStr;
 }
-
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
+
+
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
-  // add your code here
-  return str;
+  let ansStr = "";
```

Comment:
Same as above. Also, I like that you are returning `ansStr` from this function which is a good example of immutability. 👍

Reply:
if(num < 0) return "";   same as above xD... I am just a lazy sometimes xDDDDDDD will fix it soon!